tests/test_sniffers.py

```diff
@@ -51,7 +52,6 @@
     'valpo',
```

Comment:
Can you remove `aurora` as well? Per your findings in #15, it's been replaced with a different server which is _not_ a false-negative, but is indeed correctly detected as a GP portal/gateway server.